### PR TITLE
Updating CI PR triggers to only apply to the one branch

### PR DESCRIPTION
**Pull request by MSLaguana**
CI was triggering on PRs to all branches, not just the one branch.

### CODE REVIEW

**Comment by MSLaguana** [comment]
@dotnet-bot test ci please

**Comment by dilijev** [comment]
See: https://github.com/Microsoft/ChakraCore/commit/2e93ca1ce64dbaf68cca1136f69481fc4b59e6ab

File: netci.groovy [hunk 1]
```diff
@@ -117,8 +114,9 @@ def DailyBuildTaskSetup = { newJob, isPR, triggerName, groupRegex ->
     // The addition of triggers makes the job non-default in GitHub.
     if (isPR) {
         def triggerRegex = "(${dailyRegex}|${groupRegex}|${triggerName})"
-        Utilities.addGithubPRTrigger(newJob,
+        Utilities.addGithubPRTriggerForBranch(newJob,
             triggerName, // GitHub task name
+            branch,
```
Comment (MSLaguana) [inline netci.groovy]: Whoops, wrong argument order here.

Also after talking with @dilijev, we may as well just backport the version from the 1.4 branch in its whole.

**Comment by dilijev** [comment]
Also in your new PR you should reference the issue you're fixing so it's automatically closed once the change hits master.

**Comment by dilijev** [comment]
Replaced by #2887

**Pull request closed without merging**